plugins/inputs/modbus/README.md

```diff
@@ -38,6 +38,18 @@ Registers via Modbus TCP or Modbus RTU/ASCII.
   # stop_bits = 1
   # transmission_mode = "RTU"
 
+  ## Trace the connection to the modbus device as debug messages
+  # trace_connection = false
+
+  ## Enable workarounds required by some devices to work correctly
+  # workarounds = {
+      ## Pause between read requests sent to the device. This might be necessary for (slow) serial devices.
+      # pause_between_requests = "0ms"
+      ## Close the connection after every gather cycle. Usually the plugin closes the connection after a certain
+      ## idle-timeout, however, if you query a device with limited simultaneous connectivity (e.g. serial devices)
+      ## from multiple instances you might want to only stay connected during gather and disconnect afterwards.
+      # close_connection_after_gather = false
+	# }
```

Comment:
```suggestion
  # }
```

Reply:
if i remember correctly "[inputs.modbus.workarounds]" only worked at the end of a "[[inputs.modbus]]" section.